AdyenActions/Components/3DS2/Action handlers/3DS2+Delegated Authentication/ThreeDS2PlusDACoreActionHandler.swift

```diff
@@ -372,7 +376,8 @@ internal typealias VoidHandler = () -> Void
             return AdyenAuthentication.AuthenticationService(
                 passKeyConfiguration: .init(
                     relyingPartyIdentifier: delegatedAuthenticationConfiguration.relyingPartyIdentifier,
-                    displayName: cardNumber ?? Bundle.main.displayName
+                    displayName: cardNumber ?? Bundle.main.displayName,
+                    consecutiveApprovalCancellationsLimit: 2
```

Comment:
this number 2 is used a few times. if they refer to the same number, could you put it on a constant?